Resize preference dialog

ohuang12 opened the pull request:
Fix for issue [#2559](https://github.com/JabRef/jabref/issues/2559).

- [x] Change in CHANGELOG.md described
- [ ] Tests created for changes
- [ ] Screenshots added (for bigger UI changes)
- [x] Manually tested changed features in running JabRef
- [ ] Check documentation status (Issue created for outdated help page at [help.jabref.org](https://github.com/JabRef/help.jabref.org/issues)?)
- [ ] If you changed the localization: Did you run `gradle localizationUpdate`?

ohuang12 committed:
Add scroll bars to the preference dialog

ohuang12 committed:
Add changelog entry

TRvanRossum reviewed (approved):
Code looks nice to me :+1:  I do wonder whether or not a horizontal scrollbar is necessary, but it can not hurt I guess.

Siedlerchr committed:
Set POSIX filepermissions to 644 (#2637)

* Set POSIX filepermissions to 644
Fix for #2635

* Add changelog and add missing dots

tobiasdiez committed:
Improve perfomance of opening a database that still used the old groups format (#2636)

* Apply some post actions before displaying the file

* Add changes in JabRefGUI

stefan-kolb committed:
Add link to logo creator

stefan-kolb committed:
Issuestats.com is down

stefan-kolb committed:
Donate to JabRef!

koppor committed:
Update dependencies

ohuang12 committed:
Add option to copy title of bibentries (#2639)

* Add copy title option in right click dropdown

* Add localization entries for copy title

* Add changelog entry

* Add missing localization entries

Fix the localization tests by adding several missing strings in the localization files
Use the menu localization file for the copy title menu item instead of the general localization file

* Use stream to collect titles

* Use more sensical naming for variables

Fix incorrect JavaDoc on getTitle

stefan-kolb committed:
Improve issue reporting a bit #2573 (#2641)
